universal-editor-product-picker/src/universal-editor-ui-1/web-src/src/components/ProductField.js

```diff
@@ -0,0 +1,155 @@
+/*
+ * <license header>
+ */
+
+import React, { useState, useEffect } from "react";
+import { attach } from "@adobe/uix-guest";
+import {
+  Provider,
+  defaultTheme,
+  ActionButton,
+  LabeledValue,
+  Tooltip,
+  TooltipTrigger,
+  useDragAndDrop,
+  Item,
+  Grid,
+  ListView,
+
+} from "@adobe/react-spectrum";
+import { extensionId, localStorageKeySelectedProducts } from "./constants";
+import {
+  convertProductSelectionsFromStringToList,
+  convertProductSelectionsFromListToString,
+  reOrderSelections,
+} from "./selections-utils";
+import Box from '@spectrum-icons/workflow/Box';
+
+export default function () {
+  const [guestConnection, setGuestConnection] = useState();
+  const [selections, setSelections] = useState([]);
+  
+  useEffect(() => {
+    (async () => {
+      const guestConnection = await attach({
+        id: extensionId,
+      });
+      setGuestConnection(guestConnection);
+
+      await guestConnection.host.field.setStyles({
```

Comment:
Need to check after merging of https://git.corp.adobe.com/CQ/universal-editor/pull/799